core/src/main/java/org/infinispan/container/entries/VersionedRepeatableReadEntry.java

```diff
@@ -21,12 +21,12 @@
  * @author Manik Surtani
  * @since 5.1
  */
-public class ClusteredRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
+public class VersionedRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
```

Comment:
I wonder if we need this class at all, since it does not hold any data, or if we should just push the methods to `RepeatableReadEntry`.